library/src/main/scala/scala/scalajs/js/typedarray/package.scala

```diff
@@ -170,8 +169,8 @@ package object typedarray {
 
   @inline private def typedArray2arrayImpl[
       @specialized(Byte, Short, Int, Float, Double) T,
-      Repr <: TypedArray[T, Repr]](
-      array: Repr, dest: scala.Array[T]): scala.Array[T] = {
+      Repr <: TypedArray[T, Repr]](array: Repr,
+      dest: scala.Array[T]): scala.Array[T] = {
```

Comment:
No, this is an undesirable (but barely tolerated) change: if the type argument list spans more than one line, it should not share a line with term parameters. The term parameters should start on their own line. This is filed as https://github.com/olafurpg/scalafmt/issues/504